menpo/image/masked.py

```diff
@@ -125,7 +125,7 @@ def init_blank(cls, shape, n_channels=1, fill=0, dtype=np.float, mask=None):
         return cls(pixels, copy=False, mask=mask)
 
     @classmethod
-    def init_from_rolled_channels(cls, pixels, mask=None):
+    def init_from_channels_at_back(cls, pixels, mask=None):
```

Comment:
Can we deprecate this rather than just replacing it? So re-add it and deprecate it in the same manner as `rolled_channels` above (but obviously referencing `init_from_channels_at_back`)